apps/governance/src/routes/staking/home/validator-tables/consensus-validators-table.tsx

```diff
@@ -131,20 +131,25 @@ export const ConsensusValidatorsTable = ({
 
   const gridRef = useRef<AgGridReact | null>(null);
 
+  const thirdOfTotalStake = useMemo(
+    () => new BigNumber(totalStake).dividedBy(3),
+    [totalStake]
+  );
+
   const nodes = useMemo(() => {
     if (!data) return [];
     let canonisedNodes = data
       .sort((a, b) => {
-        const aVotingPower = new BigNumber(a.rankingScore.votingPower);
-        const bVotingPower = new BigNumber(b.rankingScore.votingPower);
-        return bVotingPower.minus(aVotingPower).toNumber();
+        const aStakedTotal = new BigNumber(a.stakedTotal);
+        const bStakedTotal = new BigNumber(b.stakedTotal);
+        return bStakedTotal.minus(aStakedTotal).toNumber();
```

Comment:
non-blocking: instead of casing to number, which can effect in uncontrolled rounding,  I would use here compareTo method from [Bignumber](https://mikemcl.github.io/bignumber.js/#cmp)